tests/unit/moduleapi/misc.tcl

```diff
@@ -86,4 +86,9 @@ start_server {tags {"modules"}} {
         set info [r test.clientinfo]
         assert { [dict get $info flags] == "${ssl_flag}::tracking::" }
     }
+
+    test {test detached thread safe cnotext} {
+        r test.log_tsctx "info" "Test message"
+        wait_for_log_messages 0 {"*<misc> Test message*"} 0 10 100
```

Comment:
you can use `verify_log_message` rather than `wait_for_log_messages` since you know the previous command already finished.